playbooks/roles/vhost/tasks/main.yml

```diff
@@ -39,6 +48,17 @@
     group: root
     mode: 0644
   when: COMMON_OBJECT_STORE_LOG_SYNC
+        and (ansible_distribution_release == 'precise' or ansible_distribution_release == 'trusty')
```

Comment:
@Agrendalath this only works for precise and trusty? and systemd is the solution for xenial?

Reply:
I'm just wondering what happens in the future, when it's neither of them precise trusty nor xenial the ones we are using.

Reply:
What happens with the future versions? we are not gonna be covering them if we limit to precise, trusty and xenial

Reply:
[Neither](https://github.com/open-craft/configuration/blob/agrendalath/hawthorn.1-xenial-log-sync-on-exit/playbooks/roles/supervisor/tasks/main.yml#L137) will supervisor work for [`bionic`](http://releases.ubuntu.com/18.04/). It's hardcoded there as well, but I agree it would be more convenient to make it more generic.

What do you think about:
```yaml
- name: Force logrotate on syslog stop
  lineinfile:
    path: /etc/systemd/system/syslog.service
    line: "ExecStop={{ COMMON_OBJECT_STORE_LOG_SYNC_ON_EXIT }}"
    insertafter: "^ExecStart=.*$"
  when: COMMON_OBJECT_STORE_LOG_SYNC 
        and not (ansible_distribution_release == 'precise' or ansible_distribution_release == 'trusty')
```
?

Reply:
@Agrendalath sounds good to me

Reply:
@rocioar, changed!